react-toggle-switch/my-app/src/Toggle.tsx

```diff
@@ -0,0 +1,16 @@
+import './Toggle.css';
+type Props = {
+  onSwitchClick: () => void;
+  toggleClass: string;
+};
+
+export function Toggle({ onSwitchClick, toggleClass }: Props) {
+  return (
+    <div id="container">
+      <div id="switch" className={toggleClass} onClick={onSwitchClick}>
+        <div id="circle"></div>
+      </div>
+      <p id="label">{String(toggleClass).toUpperCase()}</p>
```

Comment:
This assumes that `toggleClass` is the label that you want. If you are creating this for different languages, such as Spanish, this won't work. Although it seems wasteful, it's best to just have a different variable for labels and other computed text.